lib/repl.js

```diff
@@ -402,6 +405,16 @@ function REPLServer(prompt,
         });
       } catch (e) {
         debug('parse error %j', code, e);
+        const importErrorStr = 'Cannot use import statement outside a module';
+        if (StringPrototypeIncludes(e.message, importErrorStr)
```

Comment:
We should probably also check for the error code.

Reply:
I'll double check, but I don't think this has a code as it is an error thrown by the VM